test/unit/adb-emu-commands-specs.js

```diff
@@ -256,6 +256,38 @@ describe('adb emulator commands', () => {
         mocks.adb.verify();
       });
     }));
+    describe("network speed method", withMocks({adb}, (mocks) => {
+      it("should throw exception on invalid speed", async () => {
+        await adb.networkSpeed('light').should.eventually.be.rejectedWith("Invalid network speed");
+        mocks.adb.verify();
+      });
+      let speeds = {
+        'gsm': adb.NETWORK_SPEED.GSM,
+        'scsd': adb.NETWORK_SPEED.SCSD,
+        'gprs': adb.NETWORK_SPEED.GPRS,
+        'edge': adb.NETWORK_SPEED.EDGE,
+        'umts': adb.NETWORK_SPEED.UMTS,
+        'hsdpa': adb.NETWORK_SPEED.HSDPA,
+        'lte': adb.NETWORK_SPEED.LTE,
+        'evdo': adb.NETWORK_SPEED.EVDO,
+        'full': adb.NETWORK_SPEED.FULL
+      };
+      for (let s of _.keys(speeds)) {
```

Comment:
This can be something like
```js
for (let [key, value] of _.pairs(speeds)) {
  it(`should set network speed(${key}) correctly`, async () => {
    // ...
    mocks.adb.expects("adbExec")
      .once().withExactArgs(["emu", "network", "speed", value])
      .returns();
    // ...
  });
}